### PR TITLE
Update and fix broken links in Core docs v4 overview

### LINKED ISSUE #1664: Fix broken links in core v4 overview page
Fix broken links in the core v4 overview page

https://liskhq.github.io/lisk-docs/lisk-core/v4/index.html

### PR DESCRIPTION
**Pull request by chris529**
closes #1664

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -49,8 +51,10 @@ Christopher Braithwaite <christopher.braithwaite@lightcurve.io>
 :url_lip37: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#chain-identifiers-1
 :url_lip51: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md
 :url_lip70: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0070.md
-:url_sdkv6: v6@lisk-sdk::index.adoc
-
+:url_lip57: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md
+// :url_sdkv6: lisk-docs::v6@lisk-sdk::index.adoc
+// :url_sdkv6: v6@lisk-sdk/index.adoc
+:url_sdkv6: lisk-sdk/v6/index.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:url_sdkv6: {docs_sdk}index.adoc
```

Comment (Tschakki) [reply modules/ROOT/pages/index.adoc]: Update `:docs_sdk: lisk-sdk::` to `:docs_sdk: v6@lisk-sdk::` as well, otherwise it will point to v5 SDK docs

**Review by Tschakki — CHANGES_REQUESTED** [review]
Links to `migration-guide` and `migrator tool` are still broken

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ Christopher Braithwaite <christopher.braithwaite@lightcurve.io>
 :v_sdk: v6.0.0 (beta)
 :page-no-previous: true
 :docs_general: ROOT::
-:docs_sdk: lisk-sdk::
+:docs sdk: v6@lisk-sdk::
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]:
```suggestion
:docs_sdk: v6@lisk-sdk::
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
the link is still broken, for some reason the underscore was removed in the attribute. Please always check that the links work as expected by previewing them in the browser, before requesting review 🙏

**Comment by chris529** [comment]
Cool...Thanks Mona! I have updated it and just checked the preview, and it works now 👍

**Review by Tschakki — APPROVED** [review]
LGTM 👍

**Pull request merged**